src/pages/post-feed/index.tsx

```diff
@@ -0,0 +1,24 @@
+import type { GetStaticProps } from 'next';
+import { serverSideTranslations } from 'next-i18next/serverSideTranslations';
+import CreatPost from './CreatPost';
+import Post from './Posts';
+import { UserType } from './usersDummyData';
+
+export const getStaticProps: GetStaticProps = async ({ locale }) => ({
+  props: {
+    ...(await serverSideTranslations(locale ?? 'en', ['common'])),
+  },
+});
+
+export default function Index() {
+  const user: UserType = {
+    id: 1,
+    userPhoto: 'https://i.pinimg.com/236x/da/c5/f8/dac5f861cf4a5d0cfbe8f310016bea28.jpg',
+  };
+  return (
+    <div>
+      <CreatPost user={user} />
+      <Post />
```

Comment:
posts list should receive its data from the parent page component, always remember that all the fetching should happen on the page component, and then you pass the data as props to child commponent

```suggestion
  const user: UserType = {
    id: 1,
    userPhoto: 'https://i.pinimg.com/236x/da/c5/f8/dac5f861cf4a5d0cfbe8f310016bea28.jpg',
  };
  const posts = [/*some posts dummy data here*/];
  return (
    <div>
      <CreatPost user={user} />
      <Posts posts={posts} />
```

Reply:
I know we should fetch data from API in the main or page component but, if I transfer the data to the page component it will be many codes there so can I do that when we have real data?

Reply:
no let's do it now, because we want to do the right thing from the start, we don't want to do the work again when we fetch the data